docs/introduction/faq.md

```diff
@@ -0,0 +1,95 @@
+# FAQ
+
+## Overview
+
+### How do I get Atoms?
+
+If you participated in the fundraiser, you can check your suggested atom balance at [fundraiser.cosmos.network](https://fundraiser.cosmos.network).
+If not, you must wait until the [Cosmos Network launches](https://cosmos.network/roadmap) and Atoms are traded on exchanges.
+
+### Are Atoms listed on exchanges?
+
+No. The Cosmos Network mainnet has not yet launched, which means Atoms are _not_ on exchanges yet. $CMOS and $ATOM tokens are _not_ Cosmos Network native tokens.
+
+### How do I participate in the fundraiser?
+
+The [fundraiser](https://fundraiser.cosmos.network) is closed. The Interchain Foundation raised funds from private individuals and has hosted a public fundraising event on which ended on April 6, 2017. Both $ETH and $BTC were accepted in the fundraiser. The security of the fundraising process has been vetted extremely carefully.
+
+### What is the initial allocation of Atoms?
+
+As a public, decentralized network, the allocation of Atoms is decided by those who run the software for the Cosmos Hub. To faciliate a decision, we are creating a Swiss non-profit, the [Interchain Foundation](https://interchain.io), which is responsible for co-ordinating fundraising and allocating funds to get the network off the ground. The foundation will suggest a allocation of Atoms according to the results of the fundraiser. Users will ultimately decide the distribution for themselves when they run the software.
+
+The Interchain Foundation will suggest that 5% of the Atoms go to its initial donors, 10% go to the Interchain Foundation, 10% go to the company developing most of the software, and the remaining 75% to be distributed according to the results of the private and public fundraisers.
+
+### What is the team developing the Cosmos Network?
+
+The Cosmos Network is the first project being funded by the Interchain Foundation. Its development is led primarily by the [Tendermint team](https://tendermint.com/about).
+
+### What's the difference between Tendermint, the Cosmos Network, and the Cosmos Hub?
+
+* [Tendermint](https://tendermint.com) is a general purpose blockchain engine that uses a Byzantine-fault tolerant consensus protocol and allows applications to be written in any programming language.
+* The Cosmos Network is a heterogenous network of Proof-of-Stake blockchains that can interoperate with one-another.
+* The Cosmos Hub is the first Proof-of-Stake blockchain to be launched by the Cosmos Network; it uses Tendermint consensus, contains a built in governance protocol, and serves as co-ordinater for interoperability between other blockchains.
+* Atoms are the native cryptocurrency of the Cosmos Hub. Atoms are necessary for participating in the consensus protocol and transacting on the network.
```

Comment:
Maybe:

"Atoms are necessary for participating in consensus as well as securing and transacting on the network."